exp/internal/webhooks/machinepool.go

```diff
@@ -122,6 +118,11 @@ func (webhook *MachinePool) Default(ctx context.Context, obj runtime.Object) err
 		normalizedVersion := "v" + *m.Spec.Template.Spec.Version
 		m.Spec.Template.Spec.Version = &normalizedVersion
 	}
+
+	if m.Spec.Template.Spec.MinReadySeconds == nil {
```

Comment:
Q: can we drop this now that we have // +default=0 on the field definition?

Reply:
We need to keep it because the `+default=0` isn't used.